### PR TITLE
[nodejs] Updates to query results

### PR DESCRIPTION
**Pull request by chejennifer**
- add a `dcLink` field that links to the datacommons explore page
- enable ranking results and return place names in the data csv
- hide map results

example for the query "which counties in california have the largest population"
https://paste.googleplex.com/5076381028319232

### CODE REVIEW

File: static/nodejs_server/types.ts [hunk 1]
```diff
@@ -41,4 +41,6 @@ export interface TileResult {
     value: number;
     date: string;
   };
+  // The link to the data commons explore page for this result.
+  dcLink?: string;
```
Comment (pradh) [inline static/nodejs_server/types.ts:45]: nit: dcUrl ?  (to consistently refer to all URLs as such in this json)

File: static/src/server.ts [hunk 2]
```diff
@@ -297,25 +301,22 @@ function getBlockTileResults(
               useChartUrl
             )
           );
-          break;
-        /* TODO: foreignobject doesn't work with the svg to png converter.
-                  Need to find a different way to render an svg of the ranking
-                  table & then re-enable this.
+          break;*/
```
Comment (pradh) [inline static/src/server.ts:304]: maybe in future we might want a "client=" and decide based on that.  have a TODO?

**Review by pradh — APPROVED** [review]
Nice!

File: static/nodejs_server/types.ts (hunk 1 above)
Comment (chejennifer) [reply static/nodejs_server/types.ts]: done

File: static/src/server.ts (hunk 2 above)
Comment (chejennifer) [reply static/src/server.ts]: added the client param

**Review by pradh — APPROVED** [review]
Thanks!

**Pull request merged**